src/Microsoft.AspNetCore.Razor/CodeGenerators/CSharpTagHelperCodeRenderer.cs

```diff
@@ -742,7 +751,7 @@ private static bool IsDynamicAttributeValue(Chunk attributeValueChunk)
             return false;
         }
 
-        private static bool TryGetPlainTextValue(Chunk chunk, out string plainText)
+        internal static bool TryGetPlainTextValue(Chunk chunk, out string plainText)
```

Comment:
The choices are `private`, the occasional `protected`, `public` or the rare `// internal for testing`. Pick from that list -- `public` here.